### PR TITLE
Fix the TTA issue related to PadListCollate logic

### PR DESCRIPTION
**Pull request by Nic-Ma**
Fixes https://github.com/Project-MONAI/tutorials/issues/533

### Description
This PR fixed the inverse issue in `TTA` module when we pad the input data in DataLoader.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: monai/transforms/inverse_batch_transform.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def __init__(
         self.detach = detach
         self.pad_batch = pad_batch
         self.fill_value = fill_value
-        self.pad_collation_used = loader.collate_fn.__doc__ == pad_list_data_collate.__doc__
+        self.pad_collation_used = loader.collate_fn.__name__ == pad_list_data_collate.__name__
```
Comment (wyli) [inline monai/transforms/inverse_batch_transform.py:91]: the names may conflict, comparing the documentation seems to be a better workaround here...

**Review by wyli — COMMENTED** [review]
thanks for the quick update!

File: monai/transforms/inverse_batch_transform.py (hunk 1 above)
Comment (Nic-Ma) [reply monai/transforms/inverse_batch_transform.py]: Thanks for the reminder, updated in the latest commit.

**Review by wyli — APPROVED** [review]

**Pull request merged**